src/components/SpeciesIdentification/SpeciesIdentification.scss

```diff
@@ -0,0 +1,50 @@
+.filter-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+}
+
+.filter-item {
+    flex: 1;
+    border: 2px solid black;
+    margin-right: 50px;
+    margin-bottom: 100px;
+}
+
+.info-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    border: 2px solid black;
+}
+
+.info-item {
+    flex: 1;
+    border: 2px solid black;
+    justify-content: center;
+    align-content: center;
+}
+
+.info-img {
+    width: 200px;
+    height: 200px;
+    align-self: flex-end;
+    border: none;
+}
+
+.sub-header,
+.main-header {
+    text-align: center;
+}
+
+@media(max-width: 430px) {
+    .filter-container {
+        flex-direction: column;
+    }
+}
+
+@media (min-width: 431px) {
+    .float-container {
+        flex-direction: row;
+    }
+}
```

Comment:
Please can we add a single extra line to the end of each file?

Reply:
added